### PR TITLE
Update files to follow latest amazon linux changes

### PR DESCRIPTION
**Pull request by ng-htec**
update: init.yaml
update: variables.tf. instance type changed, cpu limit changed

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -35,7 +35,7 @@ variable "subnet_ids" {
 
 variable "instance_type" {
   type        = string
-  default     = "t3.nano"
+  default     = "t2.micro"
```
Comment (jzlatanovic) [inline variables.tf]: Set t3 instance type as default, avoid older instance types such as t2

**Review by jzlatanovic — COMMENTED** [review]
Please check the TFSec findings

**Comment by ng-htec** [comment]
> Please check the TFSec findings

Inbound from outside is necessary but they are able to modify it and allow access from only certain ip/subnet

Resource issue has been updated and limited

**Review by jzlatanovic — APPROVED** [review]
Splendid 😄

**Pull request merged**